docs/use-cases/Intel-Discrete-GPU-passthrough-and-Kata.md

```diff
@@ -0,0 +1,278 @@
+```bash
+$ sudo vim /etc/default/grub
+```
+
+2. At the end of the GRUB_CMDLINE_LINUX_DEFAULT append the below line:
+
+`intel_iommu=on iommu=pt i915.max_vfs=63 i915.enable_iaf=0`
+
+3. Update grub as per OS distribution:
+
+For Ubuntu:
+```bash
+$ sudo update-grub
+```
+
+For CentOS/RHEL:
+```bash
+$ sudo grub2-mkconfig -o /boot/grub2/grub.cfg 
+``` 
+
+4. Reboot the system
+```bash
+$ sudo reboot
+```
+
+## Install and configure Kata Containers
+
+To use this feature, you need Kata version 1.3.0 or above.
+Follow the [Kata Containers setup instructions](../install/README.md)
+to install the latest version of Kata.
+
+To use large BARs devices (for example, NVIDIA Tesla P100), you need Kata version 1.11.0 or above.
+
+In order to pass a GPU to a Kata Container, you need to enable the `hotplug_vfio_on_root_bus`
+configuration in the Kata `configuration.toml` file as shown below.
+
+```bash
+$ sudo sed -i -e 's/^# *\(hotplug_vfio_on_root_bus\).*=.*$/\1 = true/g' /usr/share/defaults/kata-containers/configuration.toml
+```
+
+Make sure you are using the `q35` machine type by verifying `machine_type = "q35"` is
+set in the `configuration.toml`. Make sure `pcie_root_port` is set to a positive value.
+
+After making the above changes, configuration in the `configuration.toml` should look like this:
+```
+machine_type = "q35"
+
+hotplug_vfio_on_root_bus = true
+pcie_root_port = 1
+```
+
+## GPU passthrough with Kata Containers
+
+Use the following steps to pass an Intel discrete GPU  with Kata:
+
+1. Find the Bus-Device-Function (BDF) for GPU device:
+
+   ```bash
+   $ sudo lspci -nn -D | grep Display
+     0000:29:00.0 Display controller [0380]: Intel Corporation Ponte Vecchio 1T [8086:0bd5] (rev 2f)
+     0000:3a:00.0 Display controller [0380]: Intel Corporation Ponte Vecchio 1T [8086:0bd5] (rev 2f)
+     0000:9a:00.0 Display controller [0380]: Intel Corporation Ponte Vecchio 1T [8086:0bd5] (rev 2f)
+     0000:ca:00.0 Display controller [0380]: Intel Corporation Ponte Vecchio 1T [8086:0bd5] (rev 2f)
+   ```
+
+   Run the previous command to determine the BDF for the GPU device on host.<br/>
+   From the previous output, PCI address `0000:29:00.0` is assigned to the hardware GPU device.<br/>
+   We choose this BDF to use it later to unbind the GPU device from the host for the purpose of demonstration.<br/>
+
+2. Find the IOMMU group for the GPU device:
+
+   ```bash
+   $ BDF="0000:29:00.0"
+   $ readlink -e /sys/bus/pci/devices/$BDF/iommu_group
+   /sys/kernel/iommu_groups/27
+   ```
+
+   The previous output shows that the GPU belongs to IOMMU group 27.
+
+3. Bind the GPU to the `vfio-pci` device driver:
+
+   ```bash
+   $ BDF="0000:29:00.0"
+   $ DEV="/sys/bus/pci/devices/$BDF"
+   $ echo "vfio-pci" | sudo tee "$DEV"/driver_override
+   $ echo $BDF | sudo tee "$DEV"/driver/unbind
+   $ echo "$BDF" | sudo tee "/sys/bus/pci/drivers_probe"
+   ```
+
+   After you run the previous commands, the GPU is  bound to `vfio-pci` driver.<br/>
```

Comment:
```suggestion
   After you run the previous commands, the GPU is bound to `vfio-pci` driver.  
```

Reply:
You can just use two spaces at the end of a line in order to force a line break.